dbt_subprojects/daily_spellbook/models/_sector/labels/addresses/bridge/identifiers/labels_bridges_near_native.sql

```diff
@@ -0,0 +1,20 @@
+{{config(
+        materialized = 'table',
+        alias = 'labels_bridges_near_native',
+        post_hook='{{ expose_spells(\'["near"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["Sector920"]\') }}')}}
```

Comment:
```suggestion
        materialized = 'table',
        tags=['static'],
        schema='labels',
        alias = 'bridges_near_native',
)}}
```

Reply:
we use static tag in these hardcoded files to avoid running in prod over and over when nothing changes.